### ISSUE TITLE
Social sharing icons not singularly read by Screenreader in browse mode

### ISSUE THREAD

**Issue by angelogti** [swiss/styleguide#635]
When using the NVDA in browse mode and selecting the social sharing icons, all links are read at once and the single links cannot be selected. 
https://swiss.github.io/styleguide/en/functions.html#04-functions-02-social-sharing

![image](https://user-images.githubusercontent.com/34123685/34714701-8def5cf4-f52a-11e7-8470-5beebea78483.png)

in the report they suggest the following: 
![image](https://user-images.githubusercontent.com/34123685/34714756-b9a0209a-f52a-11e7-9854-352c38e61430.png)

but I imagine that the issue could be solved with issue
